### PR TITLE
feat: quick compare

### PR DESCRIPTION
**Pull request by QizhengMo**

### CODE REVIEW

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/service/ReplayCaseTransmitService.java [hunk 1]
```diff
@@ -229,6 +232,11 @@ void updateSendResult(ReplayActionCaseItem caseItem, CaseSendStatusType sendStat
             caseItem.setTargetResultId(StringUtils.EMPTY);
         }
         caseItem.setSendStatus(sendStatusType.getValue());
+        if (sendStatusType == CaseSendStatusType.SUCCESS) {
+            AsyncCompareCaseTaskRunnable compareTask = new AsyncCompareCaseTaskRunnable(caseItem);
+            compareExecutorService.execute(compareTask);
+            LOGGER.info("Async compare task distributed, case id: {}", caseItem.getId());
+        }
         if (sendStatusType == CaseSendStatusType.SUCCESS && replayResultComparer.compare(caseItem, true)) {
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/service/ReplayCaseTransmitService.java:240]: The original logic has not been removed, and it seems to have been compared twice

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/service/report/QueryReplayMsgService.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+@Slf4j
+@Component
+public class QueryReplayMsgService {
+    @Resource
+    private ReplayCompareResultRepositoryImpl replayCompareResultRepository;
+    private static final CompareSDK COMPARE_INSTANCE = new CompareSDK();
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/service/report/QueryReplayMsgService.java:32]: For the scattered use of CompareSDK, is it possible to add a method to the original DefaultReplayResultComparer?

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/service/report/QueryReplayMsgService.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    public QueryDiffMsgByIdResponseType queryDiffMsgById(String id) {
+        QueryDiffMsgByIdResponseType response = new QueryDiffMsgByIdResponseType();
+
+        ReplayCompareResult compareResultBo = replayCompareResultRepository.queryCompareResultsById(id);
+
+        // bo may contain only quick compare result, need to fill log entities into BO
+        if (DiffResultCode.COMPARED_WITH_DIFFERENCE == compareResultBo.getDiffResultCode()
+                && CollectionUtils.isEmpty(compareResultBo.getLogs())) {
+            List<LogEntity> logs =
+                    COMPARE_INSTANCE.compare(compareResultBo.getBaseMsg(), compareResultBo.getTestMsg()).getLogs();
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/service/report/QueryReplayMsgService.java:44]: The comparison configuration of the previous plan has not been passed in

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/dao/mongodb/ReplayCompareResultRepositoryImpl.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @Override
+    public boolean save(List<ReplayCompareResultCollection> itemList) {
+        mongoTemplate.insertAll(itemList);
+        return true;
+    }
+
+    @Override
+    public boolean save(ReplayCompareResultCollection item) {
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/dao/mongodb/ReplayCompareResultRepositoryImpl.java:38]: Expose the data entity of the dao layer to the service layer

**Review by coryhh — APPROVED** [review]

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/CompareConfigService.java [hunk 5]
```diff
@@ -152,8 +152,13 @@ private final static class GenericResponseType<T> {
     }
 
     public ReplayComparisonConfig loadConfig(ReplayActionItem actionItem) {
+        return this.loadConfig(actionItem.getId());
+    }
+
+
+    public ReplayComparisonConfig loadConfig(String actionItemId) {
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/CompareConfigService.java:159]: private?

Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/CompareConfigService.java:159]: private?

**Review by coryhh — APPROVED** [review]

**Review by binyu1005 — APPROVED** [review]

**Pull request merged**